R/resample_draws.R

```diff
@@ -0,0 +1,176 @@
+#' Resample Draws Objects
+#'
+#' Resample [`draws`] objects according to given weights obtained,
+#' for instance, through importance sampling.
+#'
+#' @template args-methods-x
+#' @param weights A vector of positive weights of length equal to the
+#' number of draws in `draws` (as obtained by method [`ndraws`]).
+#' Draws with higher weights will be selected more often, usually
+#' roughly proportional to their weights. How exactly the weights are handled
+#' depends on the `method` argument. Weights will be internally normalized
+#' if necessary.
+#' @param method Name of the resampling method being applied.
+#' Possible choices are `"simple"` for simple random resampling with
+#' replacement, `"simple_no_replace"` for simple random resampling without
+#' replacement, `"stratified"` for stratified resampling with replacement
```

Comment:
Stratified by what? I guess we - in the long run at least - need to specify what the different approaches are doing?

Reply:
This is a good point. I need to read the paper first :-D For now I just refer to the paper for details and we can change it later on.